### PR TITLE
Auto layout button

### PR DESCRIPTION
**Pull request by paulgheorghecristian**
This commit fixes the "Auto layout" button's height so that its text is shown fully. This button is found in Pro Tools->ERD Editor
autolayout

### CODE REVIEW

File: com.cubrid.common.ui/src/com/cubrid/common/ui/er/editor/ERSchemaToolBar.java [hunk 1]
```diff
@@ -409,15 +410,13 @@ private String getDBLabel() {
...
-		gdLayout.verticalSpacing = 0;
+		final GridData gridData = new GridData(SWT.FILL, SWT.FILL, true, true);
+		
 		comp.setLayout(gdLayout);
-		comp.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, false));
+		comp.setLayoutData(gridData);
 
-		final Button autoLayoutButton = new Button(comp, SWT.PUSH | SWT.CENTER);
-		autoLayoutButton.setLayoutData(CommonUITool.createGridData(1, 1, 98, 20));
+		final Button autoLayoutButton = new Button(comp, SWT.PUSH | SWT.CENTER | SWT.FILL);
```
Comment (seunghun-kim) [inline com.cubrid.common.ui/src/com/cubrid/common/ui/er/editor/ERSchemaToolBar.java]: I think this issue is related with OS's font.
In my mac, "Auto Layout" button is printed as below.
before

How about reduce the font size?
I attached some codes example about reduce the font size. 
```java
		Font font = autoLayoutButton.getFont();
		String fontName = font.getFontData()[0].name;
		int fontSize = (int)font.getFontData()[0].height;
		// If `fontSize -1` couldn't show all characters, you have to minus more number as you need.
		autoLayoutButton.setFont(new Font(getDisplay(), fontName, fontSize - 1, SWT.NONE));
```

If you have better good idea about this, notice to me anytime. Thank you. 😃

**Comment by paulgheorghecristian** [comment]
Thank you for the font idea!
Unfortunately, it doesn't quite work as expected. If I just change the size of the font, I have to reduce it so much that the button becomes too small.:
auto

In order for it to look as expected, I had to keep my code, but also integrate yours. Now it looks like this:
auto2

Please tell me your opinion about this.

File: com.cubrid.common.ui/src/com/cubrid/common/ui/er/editor/ERSchemaToolBar.java [hunk 2]
```diff
@@ -409,15 +411,17 @@ private String getDBLabel() {
...
 		comp.setLayout(gdLayout);
-		comp.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, false));
-
-		final Button autoLayoutButton = new Button(comp, SWT.PUSH | SWT.CENTER);
-		autoLayoutButton.setLayoutData(CommonUITool.createGridData(1, 1, 98, 20));
+		comp.setLayoutData(gridData);
+
+		final Button autoLayoutButton = new Button(comp, SWT.PUSH | SWT.CENTER | SWT.FILL);
+		Font font = autoLayoutButton.getFont();
+		String fontName = font.getFontData()[0].name;
```
Comment (seunghun-kim) [inline com.cubrid.common.ui/src/com/cubrid/common/ui/er/editor/ERSchemaToolBar.java]: I found my mistake. 
`FontData.name` field is platform dependent. 
As a result, `FontData.name` couldn't use in Windows platform.
So we should change `name` field to `getName()` method like below.
`String fontName = font.getFontData()[0].getName();`

File: com.cubrid.common.ui/src/com/cubrid/common/ui/er/editor/ERSchemaToolBar.java [hunk 3]
```diff
@@ -409,15 +411,17 @@ private String getDBLabel() {
...
+		int fontSize = (int)font.getFontData()[0].height;
```
Comment (seunghun-kim) [inline com.cubrid.common.ui/src/com/cubrid/common/ui/er/editor/ERSchemaToolBar.java]: `FontData.height` is same as `FontData.name` too.
We should change like below.
`int fontSize = font.getFontData()[0].getHeight();`

**Comment by paulgheorghecristian** [comment]
Ok, so I've made further changes. I don't resize the whole toolbar anymore, nor do I change the font's size. Simply by adding gdLayout.marginHeight = 0;, as you mentioned, does the job, the button now looks ok. Please tell me what are your thoughts about this.

auto

**Comment by seunghun-kim** [comment]
Your changes made good solution. 
Now, "Auto Layout" button looks good in Mac and Windows.
I'm thanks for your efforts to makes better UX. 👍

**Comment by newpcraft** [comment]
LGTM

**Pull request merged**